proto/pdpb.proto

```diff
@@ -233,6 +233,7 @@ message RegionHeartbeatRequest {
 enum ConfChangeType {
     AddNode    = 0;
     RemoveNode = 1;
+    AddLearnerNode = 2;
```

Comment:
Why change this?

Reply:
I thought pd should also support `add learner node` command.

Reply:
Yes. But not in this pr.